tests/process_monitor/test_comm_from_server.py

```diff
@@ -664,15 +664,40 @@ def test_MantarrayProcessesMonitor__check_and_handle_server_to_main_queue__handl
-    expected_da_item = "item 3"
+    expected_file_writer_item = {
+        "board_0": {
+            "main_to_instrument_comm": [],
+            "instrument_comm_to_main": [
+                {
+                    "communication_type": "log",
+                    "log_level": 20,
+                    "message": "Microcontroller Communication Process beginning teardown at 2023-02-06 23:25:37.441945",
+                },
+                {
+                    "communication_type": "log",
+                    "log_level": 20,
+                    "message": "Microcontroller Communication Process beginning teardown at 2023-02-06 23:25:37.457458",
+                },
+                {
+                    "communication_type": "log",
+                    "log_level": 20,
+                    "message": "Remaining serial data in cache: [], in buffer: [67, 85, 82, 73, 32, 66, 73, 79, 85, 2, 42, 54, 242, 92, 0, 0, 0, 0, 1, 108, 25, 24, 93, 0, 0, 0, 0, 0, 0, 0, 2, 0, 2, 0, 2, 0, 0, 0, 2, 0, 2, 0, 2, 0, 0, 0, 2, 0, 2, 0, 2, 0, 0, 128, 1, 128, 1, 128, 1, 0, 0, 128, 1, 128, 1, 128, 1, 0, 0, 128, 1, 128, 1, 128, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 128, 0, 128, 0, 128, 0, 0, 0, 128, 0, 128, 0, 128, 0, 0, 0, 128, 0, 128, 0, 128, 0, 0, 0, 0, 4, 0, 4, 0, 4, 0, 0, 0, 4, 0, 4, 0, 4, 0, 0, 0, 4, 0, 4, 0, 4, 0, 0, 128, 3, 128, 3, 128, 3, 0, 0, 128, 3, 128, 3, 128, 3, 0, 0, 128, 3, 128, 3, 128, 3, 0, 0, 0, 3, 0, 3, 0, 3, 0, 0, 0, 3, 0, 3, 0, 3, 0, 0, 0, 3, 0, 3, 0, 3, 0, 0, 128, 2, 128, 2, 128, 2, 0, 0, 128, 2, 128, 2, 128, 2, 0, 0, 128, 2, 128, 2, 128, 2, 0, 0, 0, 6, 0, 6, 0, 6, 0, 0, 0, 6, 0, 6, 0, 6, 0, 0, 0, 6, 0, 6, 0, 6, 0, 0, 128, 5, 128, 5, 128, 5, 0, 0, 128, 5, 128, 5, 128, 5, 0, 0, 128, 5, 128, 5, 128, 5, 0, 0, 0, 5, 0, 5, 0, 5, 0, 0, 0, 5, 0, 5, 0, 5, 0, 0, 0, 5, 0, 5, 0, 5, 0, 0, 128, 4, 128, 4, 128, 4, 0, 0, 128, 4, 128, 4, 128, 4, 0, 0, 128, 4, 128, 4, 128, 4, 0, 0, 0, 8, 0, 8, 0, 8, 0, 0, 0, 8, 0, 8, 0, 8, 0, 0, 0, 8, 0, 8, 0, 8, 0, 0, 128, 7, 128, 7, 128, 7, 0, 0, 128, 7, 128, 7, 128, 7, 0, 0, 128, 7, 128, 7, 128, 7, 0, 0, 0, 7, 0, 7, 0, 7, 0, 0, 0, 7, 0, 7, 0, 7, 0, 0, 0, 7, 0, 7, 0, 7, 0, 0, 128, 6, 128, 6, 128, 6, 0, 0, 128, 6, 128, 6, 128, 6, 0, 0, 128, 6, 128, 6, 128, 6, 0, 0, 0, 10, 0, 10, 0, 10, 0, 0, 0, 10, 0, 10, 0, 10, 0, 0, 0, 10, 0, 10, 0, 10, 0, 0, 128, 9, 128, 9, 128, 9, 0, 0, 128, 9, 128, 9, 128, 9, 0, 0, 128, 9, 128, 9, 128, 9, 0, 0, 0, 9, 0, 9, 0, 9, 0, 0, 0, 9, 0, 9, 0, 9, 0, 0, 0, 9, 0, 9, 0, 9, 0, 0, 128, 8, 128, 8, 128, 8, 0, 0, 128, 8, 128, 8, 128, 8, 0, 0, 128, 8, 128, 8, 128, 8, 0, 0, 0, 12, 0, 12, 0, 12, 0, 0, 0, 12, 0, 12, 0, 12, 0, 0, 0, 12, 0, 12, 0, 12, 0, 0, 128, 11, 128, 11, 128, 11, 0, 0, 128, 11, 128, 11, 128, 11, 0, 0, 128, 11, 128, 11, 128, 11, 0, 0, 0, 11, 0, 11, 0, 11, 0, 0, 0, 11, 0, 11, 0, 11, 0, 0, 0, 11, 0, 11, 0, 11, 0, 0, 128, 10, 128, 10, 128, 10, 0, 0, 128, 10, 128, 10, 128, 10, 0, 0, 128, 10, 128, 10, 128, 10, 34, 226, 12, 164]",
```

Comment:
We shouldn't be adding strings this large into tests. Also, this wouldn't need any redaction, we want to add messages here that _will_ need redaction, and maybe only a single one that doesn't